lighthouse-core/computed/network-analysis.js

```diff
@@ -12,7 +12,7 @@ const NetworkRecords = require('./network-records.js');
 class NetworkAnalysis {
   /**
    * @param {Array<LH.Artifacts.NetworkRequest>} records
-   * @return {Omit<LH.Artifacts.NetworkAnalysis, 'throughput'>}
+   * @return {StrictOmit<LH.Artifacts.NetworkAnalysis, 'throughput'>}
```

Comment:
caused by https://github.com/Microsoft/TypeScript/pull/30552. See that thread for the drama over strictness :)

Reply:
https://github.com/microsoft/TypeScript/issues/30825#issuecomment-508584855 lol

Reply:
> wow, that is some good drama. I personally found microsoft/TypeScript#30825 more enjoyable to follow :)

oh yeah, whoops, that's a way better link